onboarding/schema.py

```diff
@@ -92,10 +92,12 @@ def pick_model_fields(model, **kwargs):
 
 def complete_onboarding(request, info, password: Optional[str]) -> JustfixUser:
     with transaction.atomic():
+        print(info)
         user = JustfixUser.objects.create_user(
             username=JustfixUser.objects.generate_random_username(),
             first_name=info["first_name"],
             last_name=info["last_name"],
+            preferred_first_name=info["preferred_first_name"],
```

Comment:
So above in line 24 of this file, you incremented `FIELD_SCHEMA_VERSION`, but a downside of that is that it will essentially "reset" the onboarding process for anyone who is currently in the middle of it, resulting in confusing behavior where all the information from their previous steps would suddenly disappear.  (Which is better than the server 500'ing, but it's still not ideal.)

I _think_ that an alternative to incrementing `FIELD_SCHEMA_VERSION` may be to simply assume that `preferred_first_name` might not actually be present in `info`.  That is, changing the above line to:

```python
preferred_first_name=info.get("preferred_first_name", ""),
```

might do the trick.  It would essentially make it so that anyone currently in the midst of onboarding would be akin to a user who chose not to fill in that field.